common/common-framework/src/test/java/cn/iocoder/common/framework/util/DateUtilTest.java

```diff
@@ -0,0 +1,57 @@
+package cn.iocoder.common.framework.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+public class DateUtilTest {
+
+    @Test
+    public void testAddDate() {
+        Assert.assertNull(DateUtil.addDate(0, 0));
+        Assert.assertEquals(new Date(1_778_410_800_000L), DateUtil.addDate(
+                new Date(1_515_585_600_000L), 2, 100));
+        Date d = new Date(1_515_585_600_000L);
```

Comment:
Does this assignment (line 15) add coverage?

Reply:
No sorry this was a line i added to check something after submitting the pr but did not realise i had pushed again after this